pkgs/applications/backup/ludusavi/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, rustPlatform, fetchCrate, cmake, pkg-config, freetype, expat,libX11 ,xorg,fontconfig}:
+rustPlatform.buildRustPackage rec {
+  pname = "ludusavi";
+  version = "0.12.1";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-ipZW3p+/Pqam4sAYLXMKMibjEH0pMCDfOFAZaCqes3k=";
```

Comment:
```suggestion
    hash = "sha256-ipZW3p+/Pqam4sAYLXMKMibjEH0pMCDfOFAZaCqes3k=";
```

Reply:
`fetchCrate` does not have a `hash` input